ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleBannerAdapter.java

```diff
@@ -74,15 +80,12 @@
   private boolean mVisibility = true;
 
   VungleBannerAdapter(@NonNull String placementId, @NonNull String uniqueRequestId,
-      @NonNull AdConfig adConfig) {
+      @NonNull AdConfig adConfig, @NonNull MediationBannerAdapter mediationBannerAdapter) {
     mVungleManager = VungleManager.getInstance();
     this.placementId = placementId;
     this.uniqueRequestId = uniqueRequestId;
     this.mAdConfig = adConfig;
-  }
-
-  void setVungleListener(@Nullable VungleListener vungleListener) {
-    this.mVungleListener = new WeakReference<>(vungleListener);
+    this.mediationAdapter = mediationBannerAdapter;
```

Comment:
this will probably break our cleanup logic for banners in VungleManager#cleanLeakedBannerAdapters since we try to check if bannerAd.getAdapter() == null in order to remove active banner. if getadapter never gets gc'd since hard reference in vunglebanneradapter, we'll need to update

we will probably need to